avocado-setup.py

```diff
@@ -430,7 +430,42 @@ def env_clean():
         helper.remove_file(postscript, postscript_dir)
 
 
-def edit_mux_file(test_config_name, mux_file_path, tmp_mux_path):
+def get_mux_inject_list(output, inp_dict):
+    inj_list = []
+    for key, value in inp_dict.items():
+        if "/" not in key:
+            full_key = "/run:%s" % key
+        else:
+            full_key = "/run/%s" % ':'.join(key.rsplit("/", 1))
+        multi = False
+        if "*" in full_key:
+            st_list = full_key.split("*")
+            prefix = st_list[0]
+            suffix = st_list[1]
+            multi = True
+        for line in output.splitlines():
+            key_to_check = full_key
+            if multi:
+                if prefix in line and suffix in line and "=>" in line:
+                    key_to_check = line.split()[0]
+            key_exist = False
+            if key_to_check in line:
+                for item in inj_list:
+                    if key_to_check in item.keys():
+                        key_exist = True
+                        break
+                if not key_exist:
+                    val_to_write = value
+                    if '"' in value:
```

Comment:
Should we also include single quote `'` ?

Reply:
I think it is better if we stick to a single workaround with `"`.

Reply:
Okay, the please add it to readme as well.
Wherever you mention quotes, mention it as double-quotes maybe.